dsgrid/config/dataset_config.py

```diff
@@ -305,6 +305,12 @@ class DatasetConfigModel(DSGBaseModel):
         description="List of data tags",
         default=[],
     )
+    enable_metric_unit_conversion: bool = Field(
```

Comment:
Just reading this makes me think "metric system" or converting everything into metric units instead of ip units. Would it be clearer to say "metric dimension"? Or just "enable_unit_conversion" (since metric dimensions are the only ones with units anyway)?

Reply:
Good point. I like`enable_unit_conversion`.